common/djangoapps/course_modes/models.py

```diff
@@ -7,6 +7,7 @@
 from django.db import models
 from collections import namedtuple, defaultdict
 from django.utils.translation import ugettext_lazy as _
+from django.utils.translation import ugettext as __
```

Comment:
I don't think you should be using `ugettext` here. The Django [i18n docs](https://docs.djangoproject.com/en/1.7/topics/i18n/translation/#working-with-lazy-translation-objects) recommend that `ugettext_lazy` be used in models and utility functions so that translations occur as late as possible, to ensure that the correct locale is in effect.